src/hatti/views/table.cljs

```diff
@@ -565,7 +565,9 @@
           (when active?
             (html
              [:div.table-view
-              {:style (when prevent-scrolling-in-table-view?
+             ;; Check if prevent-scrolling-in-table-view? value is equal to true
+             ;; since it can also be null
+              {:style (when (= true prevent-scrolling-in-table-view?)
```

Comment:
I think it would be elegant if we used `true?` i.e. `(true? prevent-scrolling-in-table-view?)`